labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -79,7 +78,11 @@ object ListManipulationExercise01 {
    * Again, easy to implement using built-in functionality, but also possible to implement in your own free-style way.
    */
   def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
+    if (l.count(x => x == e) == 0) {
+      false
+    } else {
+      true
+    }
   }
```

Comment:
I'm curious why you implemented it this way--there's another builtin that you can use that's a bit simpler. But this is cool too.